controllers/nodehealthcheck_controller_test.go

```diff
@@ -815,6 +815,20 @@ var _ = Describe("Node Health Check CR", func() {
 
 		})
 
+		Context("with Node marked for excluding remediation", func() {
+			BeforeEach(func() {
+				objects = newNodes(1, 2, false, true)
+				objects = append(objects, newNodeHealthCheck())
```

Comment:
- these 2 lines can be replaced by `setupObjects(1, 2, true)`
- the test could be placed a bit better, this is between the long remediation tests with the classic and 2 escalating tests. Everything else is below the 2nd esc. test (or even below the control plane test) 🙂

Reply:
created another PR for this: #312